### ISSUE TITLE
reduce memory usage of concourse-lite vagrant box

### ISSUE THREAD

**Issue by sneaker** [concourse/concourse-lite#6]
Would it be possible to reduce the amount of memory used by the vagrant box to something more reasonable like 2 GB instead of 6 GB?

**Comment by donaldguy** [comment]
I strongly second. 6GB may be reasonable if you have 16 or 32 gigs, but I have but 8, and as soon as I started running my first build, my entire system became unresponsive

at least a default like min(33% of RAM, 6GB) would make a much better first impression for people in my situation

(or if 6 is really necessary, bail with an error in the vagrantfile )

I discovered the VM_MEMORY env var but in a reload (to test caching behavior) I forgot to include it on a vagrant up and again totally jammed my system for several minutes until I managed to turn the vm off without having to hard reset my computer (as I did the first time)

**Comment by joshuajabbour** [comment]
See the related issue at https://github.com/starkandwayne/concourse-tutorial/issues/46.

This is definitely a *huge* problem, and it's discouraging to hear concourse-lite might be designed with these defaults. I'm not sure who has 6GB spare memory sitting around on their computers, but so far this issue took down my Mac four times before I found this issue.

Attempting to use `VM_MEMORY=2048 vagrant up`) caused `VBoxHeadless` to immediately eat up 5GB+ (instead of the previous behavior I'd seen where it would start with ~2.35 and increase to >5 only as I used Concourse).

I seem to finally have gotten it under control with the standard Vagrant memory controls:

```
config.vm.provider "virtualbox" do |vb|
  vb.memory = 2048
  vb.cpus = 2
end
```

But there really should either be better defaults, or a clear warning that concourse-lite won't run on many systems without these settings being tweaked.
